Application/Frontend/components/FriendsColumn/FriendRequest.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useContext } from 'react';
+// If you're using a context to access the sender's ID, import it here
+// import { UserContext } from '../path/to/your/UserContext';
+
+const SendFriendRequest = ({ senderID }) => {
+  // If using context to get the sender's ID, you can use useContext
+  // const { senderID } = useContext(UserContext);
+
+  const [receiverID, setReceiverID] = useState('');
+  const [message, setMessage] = useState('');
+
+  const handleInputChange = (event) => {
+    setReceiverID(event.target.value);
+  };
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    if (!receiverID) {
+      setMessage('Please provide a user ID');
+      return;
+    }
+
+    try {
+      const response = await fetch('/api/add_freind_request_to_user', {
```

Comment:
Please update your endpoint names. There is a typo in this endpoint, and it is poorly formatted (please use hyphens instead).